handler/pkce/handler.go

```diff
@@ -116,12 +117,12 @@ func (c *Handler) validate(challenge, method string, client fosite.Client) error
 	case "":
 		if !c.EnablePlainChallengeMethod {
 			return errorsx.WithStack(fosite.ErrInvalidRequest.
-				WithHint("Clients must use code_challenge_method=S256, plain is not allowed.").
+				WithHintID(i18n.ErrHintInvalidPKCECodeChallengeMethod, "S256").
```

Comment:
This is no longer `WithHint("Clients must use code_challenge_method=S256, plain is not allowed.").`

Reply:
Addressed